packages/mermaid/src/diagrams/xychart/xychartDb.ts

```diff
@@ -118,11 +119,16 @@ function setYAxisRangeFromPlotData(data: number[]) {
     type: 'linear',
     title: xyChartData.yAxis.title,
     min: Math.min(prevMinValue, minValue),
-    max: Math.max(prevMaxValue, maxValue),
+    max:
+      plotType === PlotType.BAR
+        ? prevMaxValue > -Infinity
+          ? prevMaxValue + maxValue
```

Comment:
@camueller over here there is one edge case let say for this example
```
xychart-beta
      title "Basic xychart with multiple datasets"
      x-axis "Relevant categories" [category1, "category 2", category3, category4]
      bar "dogs" [40, 20, 40, 130]
      bar "cats" [20, 140, 50, 30]
      bar "birds" [130, 60, 50, 30]
```
the max value of the axis is computed as 400 ideally it should be 220 taking the values from the 2nd column
![image](https://github.com/mermaid-js/mermaid/assets/7448177/86660f3a-9016-4d0e-a007-3bc3ed805dcb)

Reply:
Using a different approach it seems to be working well.